lib/scala/pkg/src/main/scala/org/enso/pkg/Config.scala

```diff
@@ -114,7 +114,18 @@ case class Config(
 
   /** Converts the configuration into a YAML representation. */
   def toYaml: String = {
-    val node          = implicitly[YamlEncoder[Config]].encode(this)
+    val config: Config = this
+    val noDevEdition: Config =
+      if (
+        config.edition
+          .filter(_.parent.filter(p => p.toString == "0.0.0-dev").isDefined)
+          .isDefined
```

Comment:
`filter` + `isDefined` that's pretty much `exists`

Reply:
Done in 83707c4